src/renderer/lib/versions.js

```diff
@@ -18,7 +18,7 @@ function Versions (index) {
   this.load()
 }
 
-Versions.prototype.load = function () {
+Versions.prototype.load = () => {
```

Comment:
You cannot use arrow functions here since arrow functions do not bind their own `this`.

This means that inside `Versions.prototype.*` you cannot use `this`, because the arrow function (in this case) uses the global `this` and not the `Versions` one :)

I suggest you [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) reading.